src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,12 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Book;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
+
+    public Book findByAuthor(String author);
```

Comment:
the public modifier is not necessary :) also, remember to wrap the Book object in an Optional (the return vaalue would be `Optional<Book>`)  so as to avoid nullchecks in the future 😄